cdap-api/src/main/java/co/cask/cdap/api/schedule/SchedulableProgramType.java

```diff
@@ -20,5 +20,5 @@
  * Defines types of programs that can be scheduled by the Application.
  */
 public enum SchedulableProgramType {
-  WORKFLOW, MAPREDUCE, SPARK, CUSTOM_ACTION
+  WORKFLOW, MAPREDUCE, SPARK, CUSTOM_ACTION, NONE
```

Comment:
@chtyim Removed the need of `NONE` from `ScheduleProgramType`. Please review again. Thanks!